src/server.jl

```diff
@@ -230,9 +230,9 @@ function serve(fw::FileWatcher=SimpleWatcher(file_changed_callback);
     # make request handler
     req_handler = HTTP.RequestHandlerFunction(req -> serve_file(fw, req))
 
-    server = Sockets.listen(port)
-    println("✓ LiveServer listening on http://localhost:$port/ ...\n  (use CTRL+C to shut down)")
-    @async HTTP.listen(Sockets.localhost, port;
+    server = Sockets.listen(parse(IPAddr, host), port)
```

Comment:
In fact, I also prefer to use `localhost` instead of `127.0.0.1` as the default value. However, it will trigger an error here.